src/Connection.php

```diff
@@ -215,14 +215,16 @@ public function createQueryBuilder()
 	/**
 	 * Performs operation in a transaction.
 	 * @param  callable $callback function(Connection $conn): void
+	 * @return mixed
```

Comment:
This should be documented, e.g. `@return mixed value returned by callback`. Also to `@param  callable $callback function(Connection $conn): void` should be changed to `@param  callable $callback function(Connection $conn): mixed` to be consistent